Src/DependencyCollector/Shared/Implementation/DiagnosticSourceListenerBase.cs

```diff
@@ -16,13 +16,25 @@ internal abstract class DiagnosticSourceListenerBase<TContext> : IObserver<Diagn
         protected readonly TelemetryClient Client;
         protected readonly TelemetryConfiguration Configuration;
 
-        private readonly IDisposable listenerSubscription;
+        private IDisposable listenerSubscription;
         private List<IDisposable> individualSubscriptions;
 
         protected DiagnosticSourceListenerBase(TelemetryConfiguration configuration)
```

Comment:
Because the Subscribe method is required, should we make a reference to it in the constructor?
I don't know how often a developer creates a new DiagnosticSourceListener (probably not often :) ), but I wonder that they may not know to look for the Subscribe method.

Reply:
this is an internal class, it's only us who call this method and just once.  I can add the comment, though